modules/Task.js

```diff
@@ -60,14 +60,18 @@ class Task {
     this.refresh();
   }
 
-  delete(description) {
-    this.array = JSON.parse(localStorage.getItem('tasks'));
-    const index = this.array.findIndex((item) => item.description === description);
-
+  updateIndex = (index) => {
     const num = index + 1;
     for (let i = num; i < this.array.length; i += 1) {
       this.array[i].index -= 1;
     }
+  }
+
+  delete(description) {
+    this.array = JSON.parse(localStorage.getItem('tasks'));
+    const index = this.array.findIndex((item) => item.description === description);
+
+    this.updateIndex(index);
```

Comment:
- [ ] Let's make sure our `clear completed` is working as intented "removes only checked tasks" currently it removes all tasks. let's fix that

Reply:
Thanks, @Amrhub , actually I missed that part of the instructions. Thanks for pointing it out